browse-handler/java/org/vufind/solr/handler/BrowseRequestHandler.java

```diff
@@ -843,6 +750,15 @@ private int asInt(String s)
         }
     }
 
+    /*
+     *  TODO: Research question:
+     *  Convert result from HashMap to either
+     *  org.apache.solr.common.util.NamedList or
+     *  org.apache.solr.common.util.SimpleOrderedMap?
+     *  Same question for BrowseList and other resturned object.
```

Comment:
Is this TODO still needed? If so, please fix the "resturned" typo. ;-)

Reply:
Yes, I think that's an open question of whether there is a benefit to aligning with the Solr util classes. I've fixed the typo and made the comment more clear, hopefully. In the next push.